src/config/deserialize.rs

```diff
@@ -90,3 +116,23 @@ where
         .collect();
     addrs.map_err(D::Error::custom)
 }
+
+/// A serde deserialization method to deserialize a list of eth account strings into a vector of
+/// [`Address`].
```

Comment:
This `Address` is a Filecoin Address? Do you mind elaborating a bit so it is more understandable?